src/main/resources/application.yml

```diff
@@ -17,9 +17,10 @@ spring:
 
   jpa:
     properties.hibernate.default_schema: HPA
+    hibernate.naming.physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
```

Comment:
sql server is case sensitive so need strategy to be as specified in the model files